docs/docsite/rst/developing_modules_general_aci.rst

```diff
@@ -0,0 +1,1851 @@
+            aci_class='l3extOut',
+            aci_rn='out-{0}'.format(l3out),
+            module_object=l3out,
+            target_filter={'name': l3out},
+        ),
+        subclass_2=dict(
+            aci_class='l3extLNodeP',
+            aci_rn='lnodep-{0}'.format(logical_node),
+            module_object=logical_node,
+            target_filter={'name': logical_node},
+        ),
+        subclass_3=dict(
+            aci_class='l3extRsNodeL3OutAtt',
+            aci_rn='rsnodeL3OutAtt-[{0}]'.format(fabric_node),
+            module_object=fabric_node,
+            target_filter={'name': fabric_node},
+        ),
+        # Adding subclass_4 for the ipRouteP class and child classes
+        subclass_4=dict(
+            aci_class='ipRouteP',
+            aci_rn='rt-[{0}]'.format(prefix),
+            module_object=prefix,
+            target_filter={'name': prefix},
+        ),
+        child_classes=child_classes**
+    )
+
+12. aci.get_existing() should remain as is. It is used to get the existing configuration of 'ipRouteP'.
+
+13. When state is present, we need to construct a payload which will be posted to APIC. Payload takes class_config and child_config. The class_config has the main attributes. If new attributes are added in new versions of APIC, we will add that attribute to class_config only if it is assigned a value.
+
+.. code-block:: python
+
+      if state == 'present':
+        child_configs = []
+        class_config = dict(
+            descr=description,
+            ip=prefix,
+            pref=preference,
+            nameAlias=name_alias,
+        )
+        if bfd is not None:
+            class_config['rtCtrl'] = bfd
+
+        if track_policy is not None:
+            tDn = 'uni/tn-{0}/tracklist-{1}'.format(tenant, track_policy)
+            child_configs.append({'ipRsRouteTrack': {'attributes': {'tDn': tDn}}})
+
+        aci.payload(
+            aci_class='ipRouteP',
+            class_config=class_config,
+            child_configs=child_configs
+        ),
+
+
+14. The payload function is followed by get_diff(), which is used to get the difference between the proposed and existing configurations of 'ipRouteP'. Here, the aci_class is changed to the class name your module is going to manage.
+
+.. code-block:: python
+
+      #aci.get_diff(aci_class='l3extRsNodeL3OutAtt')
+      aci.get_diff(aci_class='ipRouteP')
+
+      aci.post_config()
+
+15. The end of the module does not change and generally remains as is.
+
+.. code-block:: python
+
+      elif state == 'absent':
+          aci.delete_config()
+
+      aci.exit_json()
+
+
+    if __name__ == '__main__':
+        main()
+
+.. note::
+
+  - A newline should be added at the end of the file to ensure that the file ends with a newline character, which is a good practice in Python coding.
+  - Avoid using extra spaces or tabs at the end of lines, as this can lead to syntax errors or unexpected behavior in Python.
+
+
+Testing Our Module
```

Comment:
Here we use our, which again does not really fit the style of informal explanation